### ISSUE TITLE
About latent code of  CelebA-HQ

### ISSUE THREAD

**Issue by junyizeng** [orpatashnik/StyleCLIP#116]
Thank you very much for your open source of latent code, may I ask you if the latent code you provide for CELEBA-HQ is obtained by pre-trained  stylegan2 model on CELEBA-HQ? If so, can you also provide the pre-trained model?

**Comment by piyushnags** [comment]
@junyizeng the link to these codes is available in the README under Editing via Latent Mapper. 
Train set: https://drive.google.com/file/d/1gof8kYc_gDLUT4wQlmUdAtPnQIlCO26q/view
Test set: https://drive.google.com/file/d/1j7RIfmrCoisxx3t-r-KC02Qc8barBecr/view
